doc/sphinx-guides/source/installation/oidc.rst

```diff
@@ -80,9 +116,53 @@ The Dataverse installation will automatically try to load the provider and retri
 You should see the new provider under "Other options" on the Log In page, as described in the :doc:`/user/account`
 section of the User Guide.
 
-By default, the Log In page will show the "builtin" provider, but you can adjust this via the ``:DefaultAuthProvider``
-configuration option. For details, see :doc:`config`.
-
-.. hint::
-   In contrast to our :doc:`oauth2`, you can use multiple providers by creating distinct configurations enabled by
-   the same technology and without modifying the Dataverse Software code base (standards for the win!).
+Provision via MPCONFIG
```

Comment:
Can we add a link from `installation/config.rst` to this list? Otherwise I fear people will complain that there's no single list of MPCONFIG options.

On a related notes, it's hard to find all the docs for APIs so for now at least we list where to find all the scattered pages here: https://guides.dataverse.org/en/5.13/api/intro.html#lists-of-dataverse-apis

Reply:
Done in [edef7d5](https://github.com/IQSS/dataverse/pull/9273/commits/edef7d5bd52f3625b2e15521046d97a73f999894). See https://dataverse-guide--9273.org.readthedocs.build/en/9273/installation/config.html#dataverse-auth-oidc